packages/api/test/e2e/promise-queries.spec.ts

```diff
@@ -41,57 +42,55 @@ describe.skip('e2e queries', () => {
   it('queries state for a balance', async () => {
     const balance = await api.query.balances.freeBalance(keyring.alice.address());
 
-    expect(
-      balance.isZero()
-    ).toBe(false);
```

Comment:
This test is stronger, we should keep it

Reply:
Tslint: `Property 'isZero' does not exist on type 'Codec'.`

Reply:
Ah, ok, I can just write it as a callback like I did in quite a few other places:

```
api.query.balances.freeBalance(keyring.alice.address(), balance => {
      expect(
        balance.isZero()
      ).toBe(false);
    });
```